packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/index.js

```diff
@@ -6,10 +6,13 @@ import angularTranslate from 'angular-translate';
 
 import component from './pack-migration-offers.component';
 
+import { PROMO_DISPLAY } from './pack-migration-offers.constant';
+
 const moduleName = 'ovhManagerTelecomPackMigrationOffers';
 
 angular
   .module(moduleName, [ngTranslateAsyncLoader, uiRouter, angularTranslate])
-  .component('packMigrationOffers', component);
+  .component('packMigrationOffers', component)
+  .constant('PROMO_DISPLAY', PROMO_DISPLAY);
```

Comment:
Not a good practice to have things named the same, especially if they contain different values. Moreover exposing an angular constant doesn't seem needed here